product_search/main.py

```diff
@@ -0,0 +1,53 @@
+"""Return the inventory details for an item in the product inventory"""
+
+import sys
+from product_search import (
+    validate_item,
+    inventory_details_for_user_item,
+    _style_dictionary_items_for_output,
+    return_product_details,
+)
+from data_management import access_json_data
+
+from user_input import get_item_name_from_user
+
+
+def main(filename):
```

Comment:
So if you look at line 50, you're passing in `sys.argv[1]`. So there's no need to repeat this on line 22. Simply pass in the variable `filename` as its argument.